examples/with-userbase/components/modal/index.js

```diff
@@ -0,0 +1,126 @@
+import { useState, useEffect } from 'react'
+import userbase from 'userbase-js'
+
+function LoginModal({ toggle, modalType, setUser }) {
+  const [username, setUsername] = useState()
+  const [password, setPassword] = useState()
```

Comment:
```suggestion
  const [username, setUsername] = useState('')
  const [password, setPassword] = useState('')
```

Same as https://github.com/zeit/next.js/pull/12150/files#r414132404